File: atinject-core/src/main/java/org/atinject/core/topology/dto/GetTopologyResponse.java
package org.atinject.core.topology.dto;

import java.util.List;

import org.atinject.core.websocket.dto.WebSocketResponse;

public class GetTopologyResponse extends WebSocketResponse {

    private static final long serialVersionUID = 1L;

    private List<String> urls;
    
    public GetTopologyResponse(){
        super();
    }

    public List<String> getUrls() {
        return urls;
    }

    public GetTopologyResponse setUrls(List<String> urls) {
        this.urls = urls;
        return this;
    }
    
}

File: atinject-integration/src/main/resources/webapp/scripts/atinject/topology/GetTopologyResponse.js
define(["atinject/WebSocketResponse"],
function(WebSocketResponse){
    function GetTopologyResponse() {
        this._class = "org.atinject.core.topology.dto.GetTopologyResponse"
    };

    GetTopologyResponse.prototype = new WebSocketResponse();

        GetTopologyResponse.prototype.getUrls = function() {
        return this.urls;
    };

    GetTopologyResponse.prototype.setUrls = function(urls) {
        this.urls = urls;
        return this;
    };


    
    return GetTopologyResponse;
});
